### PR TITLE
use composer

### LINKED ISSUE #400: Use composer for dependency management
this will most likely render #363 and #367 obsolete as we'll switch to the most recent Twig 1.x in the process

### PR DESCRIPTION
**Pull request by h00j**
resolves #400

### CODE REVIEW

File: inc/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+define('TINYBOARD', 'xD');
```
Comment (ctrlcctrlv) [inline inc/bootstrap.php:2]: lol

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "inc/queue.php",
+            "inc/polyfill.php",
+            "inc/functions.php"
+        ]
+    },
+    "license": "Tinyboard + vichan",
+    "authors": [
+        {
+            "name": "h00j",
+            "email": "admin@vichan.net"
```
Comment (ctrlcctrlv) [inline composer.json:35]: this is problematic per license

Comment (h00j) [reply composer.json:35]: ahh of course, what an oversight, thanks! resolved in https://github.com/vichan-devel/vichan/commit/3242e28b19dc9ff9c90383cd96167ba3c991c873

Comment (ctrlcctrlv) [reply composer.json:35]: no problem. i'm not sure if he's still alive, but when he was, @savetheinternet was extremely litigious. i remember well.

**Pull request merged**